sentry-tracing/src/converters.rs

```diff
@@ -141,9 +142,66 @@ pub fn exception_from_event<S>(event: &tracing_core::Event, ctx: Context<S>) ->
 where
     S: Subscriber + for<'a> LookupSpan<'a>,
 {
+    const DEFAULT_EXCEPTION_TYPE: &str = "Unknown";
+    const DEFAULT_ERROR_VALUE_KEY: &str = "error";
+    const DEFAULT_ERROR_TYPE_KEY: &str = "error_kind";
+
     // TODO: Exception records in Sentry need a valid type, value and full stack trace to support
     // proper grouping and issue metadata generation. tracing_core::Record does not contain sufficient
     // information for this. However, it may contain a serialized error which we can parse to emit
     // an exception record.
-    event_from_event(event, ctx)
+    let (message, mut extra) = extract_event_data(event);
+    let ty = extra
+        .remove(DEFAULT_ERROR_TYPE_KEY)
+        .map(|v| v.as_str().map(|s| s.to_owned()))
+        .flatten()
+        .unwrap_or_else(|| String::from(DEFAULT_EXCEPTION_TYPE));
+    let value = extra
+        .remove(DEFAULT_ERROR_VALUE_KEY)
+        .map(|v| v.as_str().map(|s| s.to_owned()))
+        .flatten()
+        .or_else(|| message.clone());
+    let module = event.metadata().module_path().map(|s| s.to_string());
+    let exception = Exception {
+        ty,
+        value,
+        module,
+        ..Default::default()
+    };
+    let culprit = match (event.metadata().file(), event.metadata().line()) {
+        (Some(f), Some(l)) => Some(format!("{}L{}", f, l)),
+        _ => None,
+    };
```

Comment:
I've been trying to find out what this field is for and so far the SDK folks seem to be telling me that this is an internal field that's set during processing and also that it is deprecated.  So we shouldn't set it here.
(see also https://github.com/getsentry/relay/pull/1082)

Reply:
Removed!

I tried to shoehorn the filename and line number of the span into the exception somehow, but perhaps this information will be included elsewhere? I am unsure about this, since there's no stacktrace available.

Reply:
Yes, it would be nice to have this somewhere.  But indeed without a stacktrace that seems tricky.  I'm not very familiar with the event data format, https://develop.sentry.dev/sdk/event-payloads/#other-interfaces might have info on where else could be suitable.